Gemfile

```diff
@@ -1,3 +1,5 @@
 source 'https://rubygems.org'
 
-gem 'graphql_swift_gen', path: 'Dependencies/Swift Gen'
\ No newline at end of file
+git 'https://github.com/Shopify/graphql_swift_gen' do
+  gem 'graphql_swift_gen', branch: 'task/buy-sdk-3'
```

Comment:
I think we can make this `master`, but this is the branch the sub-module is currently pointed at.

Reply:
`master` has diverged a significant bit from `task/buy-sdk-3`. I'd keep the current branch for the time being.

Reply:
Are the tests reliable enough that we could just try changing it (probably not as part of this work) and then see if it matches?

Reply:
No, they're stale, outdated, incomplete and I wouldn't trust them for a second.

Reply:
I did a bunch of work on the SDK branch that isn't in `master`.